app/grandchallenge/algorithms/static/algorithms/js/job_session.mjs

```diff
@@ -25,12 +25,15 @@ function handleJobStatus(job) {
 
     handleImageImports(jobStatus, imageInputs)
 
-    let estimatedRemainingTime = averageJobDuration;
-
     if (jobStatus === "succeeded") {
         setCardCompleteMessage(cards.job, "View Results");
-    } else if (["started", "provisioning", "provisioned", "executing", "executed", "parsing outputs"].includes(jobStatus)) {
-        setCardActiveMessage(cards.job, `Started, ${moment.duration(estimatedRemainingTime).humanize()} remaining`);
+    } else if (["started", "provisioning", "provisioned"].includes(jobStatus)) {
+        setCardActiveMessage(cards.job, `Job is queued for execution`);
+        setTimeout(function () {
+            getJobStatus(job.api_url)
+        }, Math.floor(Math.random() * timeout) + 100);
```

Comment:
I know these lines were already repeated below, but while you're in here it would be nice to refactor this by removing the duplicate lines.

Reply:
Thanks for the comment Harm!
I agree on both points. I removed the duplication and changed the card text.
Testing now.